app/controllers/book_information_controller.rb

```diff
@@ -0,0 +1,19 @@
+class BookInformationController < ApplicationController
+  def show
+    return render_errors({ isbn: 'ISBN should be numeric' }, :bad_request) unless valid_isbn?
```

Comment:
I prefer to use full_messages in errors, and not hashes.
```suggestion
    return render_errors('ISBN should be numeric' , :bad_request) unless valid_isbn?
```

Reply:
Also, if you use hashes errors:
```suggestion
    return render_errors({ isbn: 'should be numeric' }, :bad_request) unless valid_isbn?
```